From: Alec Leamas <leamas.alec@nowhere.net>
Date: Mon, 26 Dec 2022 14:46:29 +0100
Subject: [PATCH] CMakeLists: ignore non-existing NSIS.template.in.in

NSIS.template.in.in is removed when the tarball is repackaged. Remove
references to this windows-only file.

Not acceptable upstream.

Forwarded: not-needed
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2b16220..0eee5a3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2691,7 +2691,7 @@ if (MINGW)
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   )
   add_dependencies(${PACKAGE_NAME} NSIS_config)
-else ()
+elseif (EXISTS NSIS.template.in.in)
   configure_file(
     ${CMAKE_CURRENT_SOURCE_DIR}/NSIS.template.in.in
     ${CMAKE_BINARY_DIR}/NSIS.template.in COPYONLY
